### ISSUE TITLE
[3.0] Insufficient colour contrast in active breadcrumb link

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/cwp-starter-theme#114]
**Starter 3.0.x-dev**

![image](https://user-images.githubusercontent.com/5170590/56937756-9868b500-6b52-11e9-9b2b-af4a21e60e38.png)

Axe plugin error:

>Element has insufficient color contrast of 4.44 (foreground color: #6c757d, background color: #f8f9fa, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1

cc @silverstripeux

**Comment by clarkepaul** [comment]
I think the background is supposed to either be white #fff or the link colour is supposed to be darker (maybe compare to older theme).

**Comment by robbieaverill** [comment]
CWP 1.x starter theme uses background-color: #f5f5f5 and text colour #707070
